gradle/src/main/groovy/com/flurry/android/symbols/SymbolUploadPlugin.groovy

```diff
@@ -57,29 +57,30 @@ class SymbolUploadPlugin implements Plugin<Project> {
      * @param target the project to find/create the configuration in
      * @return the project's configuration container
      */
-    private SymbolUploadConfiguration getOrCreateConfig(Project target) {
-        SymbolUploadConfiguration config = target.extensions.findByType(SymbolUploadConfiguration.class)
-        if (config == null) {
-            config = target.extensions.create(CONFIGURATION_KEY, SymbolUploadConfiguration.class)
+    private static SymbolUploadConfiguration getOrCreateConfig(Project target) {
+        SymbolUploadConfiguration config = target.extensions.findByType SymbolUploadConfiguration
+        if (!config) {
+            config = target.extensions.create CONFIGURATION_KEY, SymbolUploadConfiguration
         }
-        return config
+
+        config
     }
 
-    private Map<String, String> evaluateConfig(SymbolUploadConfiguration config) {
-        Map<String, String> configValues = new HashMap<>();
+    private static Map<String, String> evaluateConfig(SymbolUploadConfiguration config) {
+        Map<String, String> configValues = new HashMap<>()
 
         if (config.apiKey) {
             configValues[API_KEY] = config.apiKey
         }
         if (config.token) {
             configValues[TOKEN] = config.useEnvVar ? System.getenv(config.token) : config.token
         }
-        configValues.put(TIMEOUT, config.uploadTimeout)
+        configValues.put TIMEOUT, config.uploadTimeout as String
 
-        if (config.configPath != null) {
-            configValues.putAll(UploadProGuardMapping.parseConfigFile(config.configPath))
+        if (config.configPath) {
+            configValues.putAll UploadProGuardMapping.parseConfigFile(config.configPath) as Map<? extends String, ? extends String>
```

Comment:
Bad, bad cast

Reply:
Similarly, the explicit cast only gets rid of the warning.